### DISCUSSION TITLE
SOLVEMP returns obsolate class of solution

### DISCUSSION THREAD

**Discussion by oravec-juraj** [yalmip/YALMIP#1095] [General discussion] [answered]
Dear Johan,

   first of all, thank you for the great job you do by developing YALMIP! The users save a great portion of work time when using YALMIP.

I just would like to kindly ask you, if you could consider updating function SOLVEMP to return a solution SOL compatible with MPT3 instead of an obsolete MPT2 toolbox. 
Particularly, the issue is that the returned solution SOL uses outdated class POLYTOPE, instead of POLYHEDRON. Of course, one may "manually" convert the solution into the POLYHEDRON class.
Anyway, I am looking forward to the next release of YALMIP.
Best regards,

        Juraj

**Comment by johanlofberg** [comment, marked as answer]
It's been on the todo-list for close to 10 years I guess...Not top priority as I don't do much parametric programming these days

**Reply by oravec-juraj** [reply]
Dear Johan, 
thank you for your quick response. Of course, I understand that this "issue" is not of the highest priority as it is quite simple to resolve it on the user's side. 
Yes, it is almost 10 years since the MPT3 toolbox has been released and announced in the ECC 2013 paper. 
Best regards,
Juraj
